### PR TITLE
Desktop: Add text drag&drop feature for seed input

### PR DESCRIPTION
**Pull request by rihardsgravis**
# Description

Allow users to drag & drop plain text seed on seed input view

## Type of change

- Bug fix (a non-breaking change which fixes an issue)

# How Has This Been Tested?

Tested on macOS and Ubuntu

# Checklist:

- [x] My code follows the style guidelines for this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: src/desktop/src/ui/components/Dropzone.js [hunk 1]
```diff
@@ -18,6 +20,11 @@ class Dropzone extends React.Component {
          * @returns {undefined}
          */
         onDrop: PropTypes.func.isRequired,
+        /** Succesfull text drop
```
Comment (rajivshah3) [inline src/desktop/src/ui/components/Dropzone.js]: `Successful`

File: src/desktop/src/ui/components/input/Seed.js [hunk 2]
```diff
@@ -123,6 +123,13 @@ class SeedInput extends React.PureComponent {
         });
     };
 
+    onTextDrop = async (seed) => {
+        if (seed.length === MAX_SEED_LENGTH) {
```
Comment (rajivshah3) [inline src/desktop/src/ui/components/input/Seed.js]: I think we should match how the validation has been done previously: https://github.com/iotaledger/trinity-wallet/blob/7500ce006b4da868aeeda54453f0ddffe1cea739/src/desktop/src/ui/components/input/Seed.js#L96

File: src/desktop/src/ui/components/input/Seed.js [hunk 3]
```diff
@@ -123,6 +123,13 @@ class SeedInput extends React.PureComponent {
         });
     };
 
+    onTextDrop = async (seed) => {
```
Comment (rajivshah3) [inline src/desktop/src/ui/components/input/Seed.js]: Is the `async` here necessary? It doesn't look like `await` is used in this function

**Review by rajivshah3 — DISMISSED** [review]

File: src/desktop/src/ui/components/Dropzone.js [hunk 4]
```diff
@@ -13,11 +15,16 @@ import css from './dropzone.scss';
  */
 class Dropzone extends React.Component {
     static propTypes = {
-        /** Succesfull file drop callback
+        /** Succesful file drop callback
          * @param {buffer} FileBuffer - Droped file content buffer
```
Comment (rajivshah3) [inline src/desktop/src/ui/components/Dropzone.js]: `Dropped`

**Review by laumair — APPROVED** [review]
LGTM!

**Pull request merged**